gather_istio.sh

```diff
@@ -188,8 +189,8 @@ function main() {
   operatorNamespace=$(oc get pods --all-namespaces -l name=istio-operator -o jsonpath="{.items[0].metadata.namespace}")
 
   inspect "ns/${operatorNamespace}"
-  inspect MutatingWebhookConfiguration
-  inspect ValidatingWebhookConfiguration
+  inspect "mutatingwebhookconfiguration/${operatorNamespace}.servicemesh-resources.maistra.io"
+  inspect "validatingwebhookconfiguration/${operatorNamespace}.servicemesh-resources.maistra.io"
```

Comment:
Would those names vary depending on the operator/operand version? Or could we expect different sets of resources based on the version?

Reply:
Not currently, but they might change in the future. I didn't want to complicate this code with label selectors or similar things (some resources don't even have labels, so we would need to use a different approach in some cases).